config/peerpods/podvm/azure-podvm-image-handler.sh

```diff
@@ -445,6 +445,88 @@ function create_azure_image_from_scratch() {
     echo "Azure image created successfully from scratch"
 }
 
+function create_signed_image_version()
+{
+    echo "Creating Azure signed image version"
+    # UEFI_SIGNATURE_DATABASE_KEY is required to be set
+    [[ -n "${UEFI_SIGNATURE_DATABASE_KEY}" ]] && local UEFI_SDB="${UEFI_SIGNATURE_DATABASE_KEY}" || error_exit "UEFI_SIGNATURE_DATABASE_KEY is empty"
+    AZURE_DEPLOYMENT_TEMPLATE=/tmp/azure-deployment-template.json
```

Comment:
if its only local lets use small letters to distinguish from global.
eg. local deployment_template = /tmp/azure-deployment-template.json